testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1161,3 +1161,35 @@ def server_secret
   xpath = "//td[contains(text(), '#{action_name}')]/ancestor::tr[contains(@class, 'list-row-even')]/td/div/button/#{button}"
   raise "xpath: #{xpath} not found" unless find(:xpath, xpath).click
 end
+
+
+# todo is this the right place?
+# common constants for SSH bootstrap with SSH key auth test
+key_filename = 'id_rsa_bootstrap-passphrase_linux.pub'
+## authorized_keys paths on the client
+auth_keys_path = '/root/.ssh/authorized_keys'
+auth_keys_sav_path = '/root/.ssh/authorized_keys.sav'
+
+When(/^I backup the SSH authorized_keys file of host "([^"]*)"$/) do |host|
+  target = get_target(host)
+  _, ret_code = target.run("cp #{auth_keys_path} #{auth_keys_sav_path}")
+  raise 'error backing up authorized_keys on host' if ret_code.nonzero?
```

Comment:
"Error", upper case

Reply:
Ok (although it seems the upper/lowercase convention is rather random in that file :) ).